### PR TITLE
Panpasco Recipe Added

### PR DESCRIPTION
**Pull request by cjandrasits**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/panpasco/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  sha256: f66a6830ddbfe3b3f5c49dcc8b3cbd00939f2a2110836ad8a408a33765d454e8
+
+build:
+  number: 0
+  noarch: generic
+  
+requirements:
+  host: 
+    - python >=3
+    - r =3.3.2
```
Comment (bgruening) [inline recipes/panpasco/meta.yaml]: please use r-base here ... are you sure you need such an old R version?

File: recipes/panpasco/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  run:
+    - gatk =3.8
```
Comment (bgruening) [inline recipes/panpasco/meta.yaml]: Can you make the pinnings less strict?

**Comment by cjandrasits** [comment]
Hi, sorry for the delay.   
Thanks for your time and comments. I made the pinnings less strict but had to test whether it influences results.   
I would prefer to leave the restrictions on tools essential for the analysis to be able to check for changes before upgrading.

**Comment by dpryan79** [comment]
Thanks @cjandrasits!

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Pull request merged**